Test/git-issues/git-issue-622$f.dfy

```diff
@@ -0,0 +1,11 @@
+// RUN: rm -f 'git-issue-622$f/git_issue_622$f.java'
+// RUN: %dafny /compile:3 /compileTarget:java /spillTargetCode:3 "%s" > "%t"
+// RUN: file 'git-issue-622$f/git_issue_622$f.java' >> "%t"
```

Comment:
`file` is not available on Windows. Is there some other way to achieve this? I guess the point is to check that the file created has this exact name.

Reply:
`file` is available on the build machine, but produces slightly different output. Unless there's another easy solution, I think it's fine to just omit this part of the test. We would still be checking that it doesn't crash.

Reply:
I'm in the middle of refactoring the tests to use xUnit to address #632, and that will mean we can use .NET runtime/standard library utilities for these kinds of operations to stay platform independent. I need perhaps another day or two before the PR is ready though.